public/locations-list-map.js

```diff
@@ -218,7 +216,10 @@ $(function() {
     const hideList = searchParams.get('hide-list') && searchParams.get('hide-list') === 'true';
 
     if (!hideList) {
-      $(".filters-list").html(createFiltersListHTML(data_by_location).join(" "));
+      // Generate and populate filter + list HTML. Once completed, swap loading and list container.
+      $('.filters-list').html(createFiltersListHTML(data_by_location).join(" "));
+      $('.locations-loading').hide();
+      $('.locations-container').show();
```

Comment:
Do we want to show the map only if the list of states is shown, or regardless of if the list of states is shown? Just want to make sure we're not nesting the display of the map within this `if (!hideList)` check inadvertently!

Reply:
I don't understand. The `hide-map` param already exists, so that's controlled independently (albeit not consistently, that too is also in #192 via https://github.com/r-pop/findthemasks/pull/192/files#diff-6e71b4786f0ea284c53245df40a9f6bdR252)

Reply:
This code to hide the loading div and show the map div is nested under a check about whether to show the states list. I was wondering if that logically made sense. It seems like showing the map, as you described in mentioning the `hide-map` param, is independent from the `hide-list` param, and therefore shouldn't be nested within `!hideList`.

Reply:
Oops! I didn't realize that the list container is separate from the map one. I see now that the loading work you added was just about the list and not about the map at all. Sorry for the confusion!